pwndbg/aglib/dt.py

```diff
@@ -111,17 +111,22 @@ def dt(
         extra = "\n".join(extra_lines)
 
         bitpos_str = "" if not bitpos else (".%i" % bitpos)
+        offset_field_str = f"+0x{offset:04x}{bitpos_str}"
 
         if obj:
-            line = "    0x%016x +0x%04x%s %-20s : %s" % (
+            line = "    0x%016x %-10s %-20s : %s" % (
                 int(obj.address) + offset,
-                offset,
-                bitpos_str,
+                offset_field_str,
                 field_name,
                 extra,
             )
         else:
-            line = "    +0x%04x%s %-20s : %s" % (offset, bitpos_str, field_name, extra)
+            line = "    %-10s %-20s : %s" % (
```

Comment:
When we are on it, can we change the % formatting here and before to use f-string literals?